docs/versioned_docs/version-v0.83.0/developers/reference/aztecjs/accounts/classes/defaults.DefaultAccountContract.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Should the entire `reference` dir be committed, or rather .gitignored?

Reply:
it shoudl've been committed, but I just removed this version anyway and the next one correctly has the reference section